### PR TITLE
Add comment with no_net_config to sample_config_runtime

### PR DESCRIPTION
**Pull request by jerryz123**
I can never remember the name of this config correctly.

### CODE REVIEW

File: deploy/sample-backup-configs/sample_config_runtime.ini [hunk 1]
```diff
@@ -15,6 +15,7 @@ spotinterruptionbehavior=terminate
 spotmaxprice=ondemand
 
 [targetconfig]
+#topology=no_net_config
```
Comment (sagark) [inline deploy/sample-backup-configs/sample_config_runtime.ini]:
```suggestion
#Set topology=no_net_config to run without a network simulation
```

**Comment by davidbiancolin** [comment]
Probably the biggest QoL improvement of the last year :P 

Though you probably should have put the setting on a different line so users could just delete the # and be on their way

**Comment by jerryz123** [comment]
> Though you probably should have put the setting on a different line so users could just delete the # and be on their way
That's what I had originally, blame @sagark

**Comment by davidbiancolin** [comment]
> > Though you probably should have put the setting on a different line so users could just delete the # and be on their way
> 
> That's what I had originally, blame @sagark

Rofl, but what i mean is:
# Text explaining how to enable no network simulation
# <Line to uncomment>

**Pull request merged**